data/sql/updates/pending_db_world/rev_1629136628346689200.sql

```diff
@@ -0,0 +1,12 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1629136628346689200');
+
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 4966;
+
+DELETE FROM `smart_scripts` WHERE (`source_type` = 0 AND `entryorguid` = 4966);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(4966, 0, 0, 1, 11, 0, 100, 0, 0, 0, 0, 0, 0, 17, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Private Hendel - On Spawn - Normal Emote'),
+(4966, 0, 1, 2, 61, 0, 100, 0, 0, 0, 0, 0, 0, 82, 2, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Private Hendel - Linked - Add NPC Flag Questgiver'),
```

Comment:
Why don't we add this flag in creature_template?

Reply:
He has the flag already, he is getting the flag removed when he becomes hostile so i think this is here just in case the flag removal from the script will remain on a respawn. ( This is from the old script, i only changed the update time on the health check )

Reply:
Ah I never checked the script part